### DISCUSSION TITLE
Google phone call history?

### DISCUSSION THREAD

**Discussion by rafnov** [FossifyOrg/General-Discussion#534] [Ideas]
I got used to and really liked Google phone calls history. It gave me overview for all calls for a particular contact/number for long period of time. In Fossify phone I am bit confused and lost when I search a number in history as it is groupped by days too, so for many days a number is repeated many times. One time would be enough, like Google does. I have many unknown numbers in calls history and sometimes they are very similiar, like 1-2 digits difference - then its hard to differ them without particular focus. Would be nice to group history only by a specific number, not separately by days.
Am I clear enough in what I need? In short it is about Google phone history grouping with Fossify call info and look&feel.

**Comment by Aga-C** [comment]
We have an open issue for grouping by numbers: https://github.com/FossifyOrg/Phone/issues/45. Is it what you'd like to have?

**Reply by rafnov** [reply]
Yes, it seem the same. Sorry for duplicate.
